specificatie/zoek-personen.yaml

```diff
@@ -0,0 +1,265 @@
+openapi: 3.0.3
+info:
+  title: Zoek Personen endpoint definitie
+  version: 2.0.0
+  contact: {}
+paths:
+  /ingeschrevenpersonen:
+    post:
+      operationId: GetIngeschrevenPersonen
+      summary: Vindt personen
+      description: |
+        Zoek personen met één van de onderstaande verplichte combinaties van parameters en vul ze evt. aan met parameters uit de andere combinaties.
+
+
+        Default krijg je personen terug die nog in leven zijn, tenzij je de inclusiefoverledenpersonen=true opgeeft.
+
+
+        Gebruik de fields parameter als je alleen specifieke velden in het antwoord wil zien, [zie functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.3.0/features/fields_extensie.feature)
+
+
+        1.  Persoon
```

Comment:
Kunnen we de titels in de beschrijving hier zoveel mogelijk gelijktrekken met de type namen in ZoekPersonenQuery (of andersom)? Hier dus "ZoekMetGeslachtsnaamEnGeboortedatum". Of deze hier noemen.

Reply:
fixed

Reply:
done